src/components/AssetsDropZone/container.jsx

```diff
@@ -0,0 +1,22 @@
+import { connect } from 'react-redux';
```

Comment:
This is just pulling the mapStateToProps, mapDispatchToProps, and connect out of AssetsDropZone/index.jsx into AssetsDropZone/container.jsx; there's no new code.